src/lib/BLE/devBLE.cpp

```diff
@@ -71,6 +72,14 @@ void BluetoothJoystickBegin()
     bleGamepad->begin(gamepadConfig);
 }
 
+static void initialize()
+{
+  registerButtonFunction(ACTION_BLE_JOYSTICK, [](){
+    connectionState = bleJoystick;
+    devicesTriggerEvent();
```

Comment:
This line is redundant, as devices.cpp will trigger the event automatically when connectionState changes. Triggering an event is pretty heavy so best to not double up.

Reply:
Ah nice.  I can also clean this up in the devwifi lib where I lifted it from.